packages/block-library/src/rss/style.scss

```diff
@@ -27,6 +27,41 @@ ul.wp-block-rss { // The ul is needed for specificity to override the reset styl
 		}
 	}
 
+	// The following rules provide class based application of user selected text
+	// decoration via block supports.
+	&.has-text-decoration-underline li.wp-block-rss__item a {
+		text-decoration: underline;
+
+		&:focus,
+		&:active {
+			text-decoration: underline;
+		}
+	}
+
+	&.has-text-decoration-line-through li.wp-block-rss__item a {
+		text-decoration: line-through;
+
+		&:focus,
+		&:active {
+			text-decoration: line-through;
+		}
+	}
+
+	&.has-text-decoration-none li.wp-block-rss__item a {
+		text-decoration: none;
+
+		&:focus,
+		&:active {
+			text-decoration: none;
+		}
+	}
+
+	& :where(a),
+	& :where(a:focus),
+	& :where(a:active) {
+		text-decoration: none;
+	}
```

Comment:
While these styles will override the link element's global text-decoration styles, all other typography styles set in _Global styles > Typography > Links_ will affect links within the RSS block, making the behaviour selective and inconsistent.

What would be ideal I guess is for the RSS block's links to inherit from (in order of lowest specificity to highest):

1. The links element styles
2. Global styles RSS block
3. Block styles

I'm thinking we could delete these lines, and, to make sure Global styles RSS block styles overwrite link elements, bump the selector in the block.json under `"typography"`, e.g., `"__experimentalSelector": ".wp-block-rss .wp-block-rss__item-title a"` 

That way, RSS block global styles will override any defaults in _Global styles > Typography > Links >  text decoration_, and the block styles are inline anyway and will take priority.

I could be overthinking it too. 

What do you think?

Reply:
> to make sure Global styles RSS block styles overwrite link elements, bump the selector in the block.json under "typography", e.g., "__experimentalSelector": ".wp-block-rss .wp-block-rss__item-title a"

The `__experimentalSelector` support flag is deprecated and shouldn't be used. Try the [Block Selectors API](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-selectors/) instead.

This approach Ramon has outlined is what I suggested in my [earlier comment](https://github.com/WordPress/gutenberg/pull/66419#discussion_r1836059224):

> For the Global Styles side of that equation, the [Block Selectors API](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-selectors/) allows custom selectors for individual features so they target the same inner elements.

Reply:
> The __experimentalSelector support flag is deprecated and shouldn't be used. Try the [Block Selectors API](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-selectors/) instead.

Oh 🤦🏻 Thanks for setting me straight on that one @aaronrobertshaw Went straight over my head. 
Carry on.